add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $title = $_POST['title'];
+    $author = $_POST['author'];
+    $year = $_POST['year'];
+
+    $query = "INSERT INTO books (title, author, year) VALUES ('$title', '$author', $year)";
```

Comment:
Observations: The SQL query is vulnerable to SQL injection attacks due to the direct insertion of user input into the query string.\n\nRecommendations: Use prepared statements with parameterized queries to prevent SQL injection. Here's an example using PDO:

```php
$stmt = $connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
$stmt->bind_param("ssi", $title, $author, $year);
$stmt->execute();
```

This approach ensures that user input is properly escaped and prevents SQL injection vulnerabilities.